set-up-point-cloud/free-trial-limits.md

```diff
@@ -35,8 +35,12 @@ Here you can find the following:
 
 ![Syskit Point - Trial Banner](../.gitbook/assets/free-trial-limits-trial-banner.png)
 
-{% hint style="warning" %}
-**Please note!**&#x20;
-
-When the trial ends, the sync option will turn off, and you will no longer be able to click the sync button, download reports, or complete other actions. 21 days after the last day of your trial, all of the data collected in Syskit Point will be erased as per the [DPA (Data Processing Addendum)](https://www.syskit.com/data-processing-addendum/). If, during those 21 days, you purchase a Syskit Point license, all of your data will remain intact.
-{% endhint %}
+## Free Trial Expiration
+
+When the trial expires, you can expect the following:
+* **You can no longer use Syskit Point**
```

Comment:
Ja bi mozda samo tu stavilla can no longer access SKP jer use znaci da ne mogu koristit a kolko sam ja skuzila ne mogu mu nio pristupit jel? al to je samo nijansa. :)

Reply:
done